sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -105,6 +108,34 @@ export class ScholasticStandingInstitutionsController extends BaseController {
     @UserToken() userToken: IInstitutionUserToken,
   ): Promise<PrimaryIdentifierAPIOutDTO> {
     try {
+      const application = await this.applicationService.getActiveApplication(
+        applicationId,
+        locationId,
+      );
+      if (!application) {
+        throw new CustomNamedError(
```

Comment:
The custom name error is not required anymore here because we are not translating the error. We can throw the unprocessible entity error directly here.

Also the try/catch can be removed as there are no custom errors thrown from `studentScholasticStandingsService.saveScholasticStandingCreateReassessment()`

The method could be simplified like this. 

```ts
 async saveScholasticStanding(
    @Param("locationId", ParseIntPipe) locationId: number,
    @Param("applicationId", ParseIntPipe) applicationId: number,
    @Body() payload: ScholasticStandingAPIInDTO,
    @UserToken() userToken: IInstitutionUserToken,
  ): Promise<PrimaryIdentifierAPIOutDTO> {
    const application = await this.applicationService.getActiveApplication(
      applicationId,
      locationId,
    );

    if (!application) {
      throw new UnprocessableEntityException(
        new ApiProcessError(
          "Application Not found or invalid current assessment or offering.",
          APPLICATION_NOT_FOUND,
        ),
      );
    }

    if (application.isArchived) {
      throw new UnprocessableEntityException(
        new ApiProcessError(
          "This application is no longer eligible to request changes.",
          APPLICATION_CHANGE_NOT_ELIGIBLE,
        ),
      );
    }
    payload.data.applicationOfferingStartDate =
      application.currentAssessment.offering.studyStartDate;
    payload.data.applicationOfferingEndDate =
      application.currentAssessment.offering.studyEndDate;
    const submissionResult =
      await this.formService.dryRunSubmission<ScholasticStanding>(
        FormNames.ReportScholasticStandingChange,
        payload.data,
      );

    if (!submissionResult.valid) {
      throw new BadRequestException("Invalid submission.");
    }

    const newStudentScholasticStanding =
      await this.studentScholasticStandingsService.saveScholasticStandingCreateReassessment(
        userToken.userId,
        application,
        submissionResult.data.data,
      );
    return { id: newStudentScholasticStanding.id };
  }
```